app/enterprise/0.35-x/deployment/installation/centos.md

```diff
@@ -4,20 +4,117 @@ title: How to Install Kong Enterprise on CentOS
+        example, if your access key is `bob-company`, your username will be `bob-company%40kong`.
+        
+        Now you can update yum with $yum update
+        You should now be able to install Kong enterprise using yum. 
+
+NOTE: The following instructions explain how to install Kong with an RPM. 
+
+3. Obtain the Kong license file in .json format and copy it to your CentOS system home directory. 
+
+## NOTE: Users with Bintray accounts should visit https://bintray.com/kong/<YOUR_REPO_NAME>/license#files to retrieve their license. Your license.json file will look something like this:
+{"license":{"signature":"LS0tLS1CRUdJTiBQR1AgTUVTU0FHRS0tLS0tClZlcnNpb246IEdudVBHIHYyCgpvd0did012TXdDSFdzMTVuUWw3dHhLK01wOTJTR0tLWVc3UU16WTBTVTVNc2toSVREWk1OTFEzVExJek1MY3dTCjA0ek1UVk1OREEwc2pRM04wOHpNalZKVHpOTE1EWk9TVTFLTXpRMVRVNHpTRXMzTjA0d056VXdUTytKWUdNUTQKR05oWW1VQ21NWEJ4Q3NDc3lMQmorTVBmOFhyWmZkNkNqVnJidmkyLzZ6THhzcitBclZtcFZWdnN1K1NiKzFhbgozcjNCeUxCZzdZOVdFL2FYQXJ0NG5lcmVpa2tZS1ozMlNlbGQvMm5iYkRzcmdlWFQzek1BQUE9PQo9b1VnSgotLS0tLUVORCBQR1AgTUVTU0FHRS0tLS0tCg=","payload":{"customer":"Test Company Inc","license_creation_date":"2017-11-08","product_subscription":"Kong Enterprise","admin_seats":"5","support_plan":"None","license_expiration_date":"2017-11-10","license_key":"00141000017ODj3AAG_a1V41000004wT0OEAU"},"version":1}}
+
+## Securely copy the license file to the CentOS system ##
+$ scp kong-se-license.json <centos username>@192.168.91.109:~
+
+4. Install Kong Enterprise (Note your version will be different based on when you obtained the rpm)
+
+$  sudo yum install kong-enterprise-edition-0.35-1.el7.noarch.rpm
+
+5. Install Postgresql 
+
+$ sudo yum install https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+
+$ sudo yum install postgresql95 postgresql95-server
+
+6. Initialize Postgresql database
+
+$ sudo /usr/pgsql-9.5/bin/postgresql95-setup initdb
+
+7. Start Postgres and Enable automatic start
+
+$ sudo systemctl enable postgresql-9.5
+$ sudo systemctl start postgresql-9.5
+
+8. Create Kong database and user. NOTE: Make sure the username and password for the Kong Database are kept safe. We have used a simple example below for illustration purposes. 
+
+## switch to postgres user ##
+$sudo -i -u postgres
+
+## launch postgresql ##
+$psql
+
+## Create kong user and database, make kong the owner of the database and set the password of the kong user to 'kong' ##
+$CREATE USER kong; CREATE DATABASE kong OWNER kong; ALTER USER kong WITH password 'kong';
+
+## exit from psql ##
+$\q
+
+## return to terminal ##
+$ exit
+
+9. Modify postgres conf to allow password authentication with md5. 
+
+## Under IPv4 local connections: replace the "ident" entry with "md5"
+$ sudo vi /var/lib/pgsql/9.5/data/pg_hba.conf 
+
+## restart postgresql ##
+$ sudo systemctl restart postgresql-9.5
+
+10. Copy the license file from your home directory to the /etc/kong directory. Kong will look for the licence here. 
```

Comment:
Instead of "Kong will look for the license", could we use something less anthropomorphic such as "Kong will use the license located here."

```suggestion
10. Copy the license file from your home directory to the `/etc/kong` directory. Kong will look for the license here. 
```